zebra-rpc/src/methods.rs

```diff
@@ -89,3 +180,9 @@ pub struct GetBlockChainInfo {
     chain: String,
     // TODO: add other fields used by lightwalletd (#3143)
 }
+
+#[derive(Debug, Eq, PartialEq, serde::Serialize, serde::Deserialize)]
+/// Response to a `sendrawtransaction` RPC request.
+///
+/// A JSON string with the transaction hash in hexadecimal.
+pub struct SentTransactionHash(String);
```

Comment:
Question: would this be better as a typed structure?

Would that need more `serde` derivations in `zebra-chain`? And do we want to make those changes?

```suggestion
pub struct SentTransactionHash(transaction::Hash);
```

Reply:
I like this change, looks like `transaction::Hash` is ready to go:
<img width="481" alt="image" src="https://user-images.githubusercontent.com/552961/156318024-2c89f97e-b426-4bca-ae52-345fea4f14be.png">

Reply:
From the snapshot tests, I think the default `serde` array serialization is an integer list of byte values, not a hexadecimal string.

So I'm happy to go with whatever is simpler here.

But if we can make a change that simplifies the remaining 9 RPCs, let's do that.

Reply:
I had an idea on how to improve it, but since it expanded the scope a bit and risks making things a bit more complex, I opened a separate PR for it (#3706). If that's a reasonable approach we can merge it into this PR first, otherwise we can merge this PR first and either tweak that PR more to merge it or just close/replace it.

Reply:
It think it's more complex to start with, but it allows the compiler to check that we're using the correct types.